tests/templates/test_subroutines/test_trotter.py

```diff
@@ -399,11 +399,19 @@ def test_convention_approx_time_evolv(self, time, n):
             qml.matrix(op2, wire_order=hamiltonian.wires),
         )
 
-    def test_queuing(self):
+    @pytest.mark.parametrize(
```

Comment:
Is it worth adding the fixture to test with both old and new op math?

Reply:
I'd say no, since we have a legacy opmath test in the test matrix now, I don't think it's necessary to do it with every CI run.